target/linux/lantiq/xrx200/base-files/etc/board.d/01_leds

```diff
@@ -33,6 +33,10 @@ zyxel,p-2812hnu-f1|\
 zyxel,p-2812hnu-f3)
 	ucidef_set_led_wlan "wifi" "wifi" "green:wlan" "phy0radio"
 	;;
+arcadyan,vrv9510kwac23)
```

Comment:
This section should be moved just before:
~~~
avm,fritz3370-rev2-hynix|\
avm,fritz3370-rev2-micron|\
avm,fritz3390)
~~~
The devices will then be sorted alphabetically.

Reply:
Done